### DISCUSSION TITLE
(suggestion) (question) iteminhand holograms, /sell gui

### DISCUSSION THREAD

**Discussion by johnsoncui02** [Zrips/CMI#1810] [Enhancement]
I was wondering where I can locate the [price] to add a $ inside of it. or maybe if i can locate where %cmi_iteminhand_worth_one% is located in the language file that would be nice.
![image](https://user-images.githubusercontent.com/23487437/58205051-829e6980-7c92-11e9-8733-1a62a73751b0.png)

About the /sell GUI. I'm suggesting we to make each individual item to be detected in the worth.yml file and it would display how much it's worth hence checking the quantity and the itemdata when placed into the sell gui.

![image](https://user-images.githubusercontent.com/23487437/58205209-e2951000-7c92-11e9-8d70-b869067ee200.png)

Add some sort of counter function to allow server owner create a timer for actionbar and such separately. It's useful for temporary commands like /tfly to show how much time they have left before the command expires.

**Comment by Zrips** [comment]
%cmi_iteminhand_worth_one% is formatted money amount, and that format comes from economy plugin itself
For sell gui, will have to dig around how could i manipulate item lore without actually changing it, so this will be on a packet level thing, so maybe at some point.

**Comment by johnsoncui02** [comment]
Update: Possibly add a chance system for attached commands as well? so there's a percentage chance to execute a random command within all of the list of commands perhaps.
